lib/herodot/configuration.rb

```diff
@@ -27,11 +27,12 @@ def worklog_file
   end
 
   def projects_directory
-    File.expand_path(@config['projects_directory'])
+    File.expand_path(@config['projects_directory'] || DEFAULT_CONFIGURATION['projects_directory'])
   end
 
   def work_times
-    @config['work_times'].map { |k, v| [k.to_sym, v.split(':').map(&:to_i)] }
+    (@config['work_times'] || DEFAULT_CONFIGURATION['work_times'])
+      .map { |k, v| [k.to_sym, v.split(':').map(&:to_i)] }
```

Comment:
How about merging the loaded config into the `DEFAULT_CONFIGURATION` if the config file exists?
Shouldn't suffice? Then you'd not need to apply the defaults in every method…

Reply:
XD too late

Reply:
But maybe we can do that another time. I just wanted to get this in now.